src/data/battler-tags.ts

```diff
@@ -217,7 +222,7 @@ export abstract class MoveRestrictionBattlerTag extends BattlerTag {
    * @param _moveId - The {@linkcode MoveId | move} being interrupted
    * @returns text to display when the move is interrupted
    */
-  interruptedText(_pokemon: Pokemon, _moveId: MoveId): string {
+  public interruptedText(_pokemon: Pokemon, _moveId: MoveId): string {
```

Comment:
```suggestion
  public getInterruptedText(_pokemon: Pokemon, _moveId: MoveId): string {
```

feels a bit weird to have selectionDeniedText being abstract, and this one not

Reply:
There are some tags (e.g. for Gorilla Tactics and Torment) that don't override this function because they are incapable of interrupting moves. If made abstract, this has to be copied into each of those tags:

```ts
override getInterruptedText(_pokemon: Pokemon, _moveId: MoveId): string {
  return "";
}
```

Should it still be made abstract? It seems more cumbersome to me that way.